mailsender.py

```diff
@@ -169,14 +169,14 @@ def email_callback(self, ch, method, properties, body):
         if send_email_message:
             email_send_id = send_email_message.send_email_id
             try:
-                print("Sending email {} to bespin-api.".format(email_send_id))
+                print(("Sending email {} to bespin-api.".format(email_send_id)))
```

Comment:
Looks like `2to3` doubled up parens for print: `print((`.

Reply:
Yeah, I know it did that and trusted it had a good reason.

Apparently it doesn't, and it just blindly adds them, via https://docs.python.org/3.0/library/2to3.html

> If this option is not given, the print fixer will surround print calls in an extra set of parentheses because it cannot differentiate between the print statement with parentheses (such as `print ("a" + "b" + "c")`) and a true function call.

Basically, 2to3 doesn't know if we're printing a tuple or calling `print()`. I'll back that out.